models/model.js

```diff
@@ -0,0 +1,10 @@
+const db = require("../db/connection")
+
+exports.selectTopics = () => {
+return db.query('SELECT * FROM topics').then(({rows : topics}) => {
+    return topics
+})
+.catch((err) => {
+    if (err) next(err);
+});
+};
```

Comment:
This catch block is in the wrong place, the promise chain ends in the controller and we would potentially miss any errors in there and cause the app to crash. This must be moved into the controller.